README.md

```diff
@@ -581,6 +581,61 @@ labels = ["LivesIn", "Visits"]
 path = "rel_examples.jsonl"
 ```
 
+#### spacy.Lemma.v1
+
+The `Lemma.v1` task lemmatizes the provided text and updates the `lemma_` attribute in the doc's tokens accordingly.
+
+```ini
+[components.llm.task]
+@llm_tasks = "spacy.Lemma.v1"
+examples = null
+```
+
+| Argument                  | Type                                    | Default                                                   | Description                                                                                                                                           |
+| ------------------------- | --------------------------------------- |-----------------------------------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `template`                | `str`                                   | [lemma.jinja](./spacy_llm/tasks/templates/lemma.jinja) | Custom prompt template to send to LLM backend. Default templates for each task are located in the `spacy_llm/tasks/templates` directory.              |
+| `examples`                | `Optional[Callable[[], Iterable[Any]]]` | `None`                                                    | Optional function that generates examples for few-shot learning.                                                                                      |
+
+`Lemma.v1` prompts the LLM to lemmatize the passed text and return the lemmatized version as a list of tokens and their
+corresponding lemma. E. g. the text 
+`I'm buying ice cream for my friends` should invoke the response
+```
+I: I
+'m: be
+buying: buy
+ice: ice
+cream: cream
+for: for
+my: my
+friends: friend
+.: .
+```
+
+If you are expecting double square brackets in your text, consider (temporarily) replacing them with other symbols, as 
+matching up lemmas to tokens will not work otherwise.
+
+If for any given text/doc instance the number of lemmas returned by the LLM doesn't match the number of tokens recognized 
```

Comment:
This probably also needs to be rephrased?

Reply:
Fixed in https://github.com/explosion/spacy-llm/pull/164/commits/b03fd2be706a6775147cd4aad923e6f94151b169.